### PR TITLE
Updated integration tests workflow to set global working directory

### LINKED ISSUE #363: Update integration tests workflow to set global `working-directory`
As of now most integrations tests workflow have a global `working-directory` set to the integration folder. 

We should standardize this to avoid confusing errors on those workflows that don't have it set.

This is missing only in the `ollama` and `unstructured` workflows.

Example workflow with default global `working-directory`.
https://github.com/deepset-ai/haystack-core-integrations/blob/5096aea315d16e2443af94b2ae636189052cd26b/.github/workflows/chroma.yml#L13-L15

### PR DESCRIPTION
**Pull request by sahusiddharth**
- fixes #363 

Standardize global working-directory for integration tests
Fixes inconsistency by setting global working-directory for ollama and unstructured workflows.

### CODE REVIEW

**Comment by sahusiddharth** [comment]
hello @vblagoje, can you please help me with the tests that are failing

**Review by masci — DISMISSED** [review]
When adding `defaults`, remember to remove the `working-directory` field from the single steps below

File: .github/workflows/ollama.yml [hunk 1]
```diff
@@ -10,6 +10,10 @@ on:
       - "integrations/ollama/**"
       - ".github/workflows/ollama.yml"
 
+defaults:
+  run:
+    working-directory: integrations/unstructured
```
[the remark this answers is not in the document]
Comment (sahusiddharth) [reply .github/workflows/ollama.yml]: sorry for the silly mistake, have changed it :)

**Comment by sahusiddharth** [comment]
Error: An error occurred trying to start process '/usr/bin/bash' with working directory '/home/runner/work/haystack-core-integrations/haystack-core-integrations/integrations/unstructured'. No such file or directory

I don't know how extra `haystack-core-integrations/` is getting introduced in the path

**Comment by anakin87** [comment]
@sahusiddharth some special steps (e.g. `docker run`...) need to run from the root directory.
I pushed a fix and am going to approve this PR.

**Review by anakin87 — APPROVED** [review]

**Pull request merged**